### DISCUSSION TITLE
Edge beta driver on linux?

### DISCUSSION THREAD

**Discussion by Tome-Jorge** [seleniumbase/SeleniumBase#967] [Ideas]
Hi!

Since edge is now available for linux in beta, would it be possible to add the option to download the edge beta driver for linux? This way one could run all tests on Linux which would be great tbh :) 
https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/

A discussion: https://stackoverflow.com/questions/67155439/how-do-i-run-the-dev-releases-of-edge-and-msedgedriver-with-python-selenium-an

Thanks you for your great work and for keeping supporting such an awesome project.

**Comment by mdmintz** [comment]
@Tome-Jorge That sounds like a reasonable request. :) Can you open an official ticket on the SeleniumBase GitHub page with all those details?

**Comment by mdmintz** [comment]
@Tome-Jorge Also, it looks like the Edge Driver for Linux is in beta. Do you know when it's going official? With Selenium 4, I've basically said that I'm waiting for the official release, as opposed to the beta in case there are major changes. I may do the same for Linux Edge Driver. Once it's out of beta for Linux, it's definitely going to be part of the official driver install scripts for SeleniumBase.

**Comment by Tome-Jorge** [comment]
@mdmintz  I am unsure of when it will leave beta, It took MS 6 months to go from dev to beta, hopefully soon. It makes sense to wait due to possible changes to the final release - and to have a stable driver. I'll definitely keep an eye out.

**Reply by mdmintz** [reply]
@Tome-Jorge In case you missed the update from a month ago, EdgeDriver (stable) has been available for Linux, and is integrated with SeleniumBase.
